docs/components.md

```diff
@@ -48,6 +48,7 @@ The distribution offers support for the following components.
 | [solace](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/solacereceiver)                           | [beta]           |
 | [splunk_hec](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/splunkhecreceiver)                    | [beta]           |
 | [sqlquery](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/sqlqueryreceiver)                       | [alpha]          |
+| [sshcheck](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/sshcheckreceiver) | alpha |
```

Comment:
Is the indentation difference just GitHub, or does this need to be fixed to match the rest of the rows?

Reply:
Intellij does weird things to markup, but it can fix it quick.